Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_Generic.yml

```diff
@@ -0,0 +1,2791 @@
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Okta v2
+                    ignorecase: true
+                accessor: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 190,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "42":
+    id: "42"
+    taskid: 51786e50-9c7b-481e-8bb3-61c809711c47
+    type: title
+    task:
+      id: 51786e50-9c7b-481e-8bb3-61c809711c47
+      version: -1
+      name: Okta
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "13"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 190,
+          "y": 790
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "43":
+    id: "43"
+    taskid: 1e94ed10-2733-4148-8982-a1b6f17c3bb6
+    type: title
+    task:
+      id: 1e94ed10-2733-4148-8982-a1b6f17c3bb6
+      version: -1
+      name: PAN-OS
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "33"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1200,
+          "y": 790
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: 4d527a3e-ea54-4f0c-84a8-eda5a4a84130
+    type: title
+    task:
+      id: 4d527a3e-ea54-4f0c-84a8-eda5a4a84130
+      version: -1
+      name: Okta
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "21"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1900,
+          "y": 790
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "45":
+    id: "45"
+    taskid: 0c346c23-69f5-4e9c-8350-2ab4c133f5fe
+    type: title
+    task:
+      id: 0c346c23-69f5-4e9c-8350-2ab4c133f5fe
+      version: -1
+      name: Related alerts from XDR
```

Comment:
need to add a check after if the XDR is enabled (and active :) )

Reply:
Done